data/dataset/mongo_example_test_dataset.yml

```diff
@@ -6,7 +6,7 @@ dataset:
       - name: customer_details
         fields:
           - name: _id
-            data_categories: [system.operations]
+            data_categories: [system.operations, user.unique_id]
```

Comment:
interesting use cases to be returning object ids. 

 I remember a conversation winter 2021 where we were talking about how it was less likely that people would put data categories on these _id fields or any ids because they weren't PII, they were really system details.

Reply:
You're right, the initial conclusion was not to handle them intentionally